src/app/src/pages/redteam/setup/components/RunOptions.tsx

```diff
@@ -144,11 +223,24 @@ export const RunOptionsContent: React.FC<RunOptionsProps> = ({
   );
 };
 
+interface RunOptionsProps extends RunOptionsBaseProps {}
+
 export const RunOptions: React.FC<RunOptionsProps> = (props) => {
   const [expanded, setExpanded] = useState(true);
+
+  const normalizedProps: RunOptionsBaseProps = {
```

Comment:
Shouldn't this be `: RunOptionsProps`?

Reply:
It could have been either, they were the same shape, the issue was that we didn't need both. Removed RunOptionsBaseProps.